webservices/resources/load.py

```diff
@@ -0,0 +1,30 @@
+from webservices.env import env
+from flask import request
+from webservices import utils
+import os
+
+es = utils.get_elasticsearch_connection()
+
+if env.get_credential('WRITE_AUTHORIZED_TOKENS'):
```

Comment:
The `get_credential` method checks the environment, so this block can be replaced with `write_cred = env.get_credential('WRITE_AUTHORIZED_TOKENS', '')`.

https://github.com/jmcarp/py-cfenv/blob/master/cfenv/__init__.py#L62-L70